scrapers/wefuckblackgirls.yml

```diff
@@ -0,0 +1,38 @@
+name: "wefuckblackgirls"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wefuckblackgirls.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="scene-title"]/h3/text()
+      Date:
+        selector: //meta[@itemprop="uploadDate"]/@content
+        postProcess:
+          - replace:
+              - regex: ([\d-]*).+
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //p[@itemprop="description"]/text()
+      Tags:
+        Name: //dl[@class="detail-list categories-list"]/dd/a/text()
+      Performers:
+        Name: 
+          selector: //div[@class="performers"]/span/text()
+          split: ", "
+      Image:
+        selector: //div[@class="player-join"]/@style
+        postProcess:
+          - replace:
+            - regex: .+\((.+)\).+
+              with: $1
+            - regex: ^
+              with: "https:"
+      Studio:
+        Name:
+          fixed: "We Fuck Black Girls"
```

Comment:
```yaml
      Studio:
        Name:
          selector: //meta[@itemprop="thumbnailUrl"]/@content
          postProcess:
            - replace:
                - regex: '^https://(?:[^\.]+\.)?([^\./]+).+'
                  with: $1
            - map:
                blackmeatwhitefeet: Black Meat White Feet
                blacksonblondes: Blacks On Blondes
                cuckoldsessions: Cuckold Sessions
                gloryhole-initiations: Gloryhole-Initiations
                gloryhole: Glory Hole
                interracialblowbang: Interracial Blow Bang
                katiethomas: Katie Thomas
                ruthblackwell: Ruth Blackwell
                springthomas: Spring Thomas
                watchingmydaughtergoblack: Watching My Daughter Go Black
                watchingmymomgoblack: Watching My Mom Go Black
                wefuckblackgirls: We Fuck Black Girls
                wifewriting: Wife Writing
                zebragirls: Zebra Girls
                #below not tested
                barbcummings: Barb Cummings
                blacksoncougars: Blacks On Cougars
                candymonroe: Candy Monroe
                dogfartbehindthescenes: Dogfart Behind The Scenes
                interracialpickups: Interracial Pickups
                theminion: The Minion
```
I used the thumbnail URL to get the actual studios.
For the last 6 i coudn't find a scene in the site to verify